lib/middleware.js

```diff
@@ -323,15 +323,15 @@ exports.mwl = function(req, rsp, next){
   // `.js` (Browserify) are actually being used to specify
   // source files
 
-  if (['js'].indexOf(ext) === -1) {
```

Comment:
`['js'].indexOf(ext) === -1` seems like an unusual way of saying `ext !== 'js'`; is there a particular reason for wording it this way?